### PR TITLE
tools/jxlinfo: improve CLI option parsing

### PR DESCRIPTION
**Pull request by thebombzen**
~~Calling "jxlinfo --help" should provide the default help
message rather than failing with "failed to read file --help."
This fixes that behavior to work as expected.~~

This patch adds --help, --verbose, and other convenience
features, such as the ability to read multiple files with
jxlinfo.

### CODE REVIEW

**Comment by jonsneyers** [comment]
Maybe allow -h too then?

**Comment by thebombzen** [comment]
`-h` and `-?` could both be added, if we desired

**Comment by mo271** [comment]
> `-h` and `-?` could both be added, if we desired

Personally, I think that `-h` and `--help` would be enough, but if you think `-?` would also be used, then feel free to add that as well!

**Comment by thebombzen** [comment]
Aight, ready for review again, after a bunch of forcepushing

File: tools/jxlinfo.c [hunk 1]
```diff
@@ -397,39 +397,65 @@ int PrintBasicInfo(FILE* file, int verbose) {
   return seen_basic_info;
 }
 
+static void print_usage(const char* name) {
+  fprintf(stderr,
+          "Usage: %s [-v] INPUT\n"
+          "  INPUT      input JPEG XL image filename\n"
```
Comment (jonsneyers) [inline tools/jxlinfo.c:403]:
```suggestion
          "  INPUT      input JPEG XL image filename(s)\n"
```

Comment (thebombzen) [reply tools/jxlinfo.c:403]: Ah right, since it now supports multiple inputs.

**Review by jonsneyers — APPROVED** [review]

**Pull request merged**